CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [Unreleased]
```

Comment:
Looks like you missed adding the link down at the bottom for `Unreleased`. Could you update it similarly to https://github.com/datajoint/pharus/commit/e6ae1bfe96562571eafc3ca9d9705c0d8a43d7af?